src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVM.cpp

```diff
@@ -246,10 +243,7 @@ real64 SinglePhaseFVM< BASE >::calculateResidualNorm( real64 const & GEOS_UNUSED
       solverBaseKernels::L2ResidualNormHelper::computeGlobalNorm( localResidualNorm[0], localResidualNormalizer[0], residualNorm );
     }
 
-    if( getLogLevel() >= 1 && logger::internal::rank == 0 )
-    {
-      std::cout << GEOS_FMT( "        ( R{} ) = ( {:4.2e} )", FlowSolverBase::coupledSolverAttributePrefix(), residualNorm );
-    }
+    GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::Convergence, GEOS_FMT( "        ( R{} ) = ( {:4.2e} )", FlowSolverBase::coupledSolverAttributePrefix(), residualNorm ));
```

Comment:
I thnk this is still an issue @arng40 . @paveltomin ?

Reply:
Yes, the line break issue for residuals output still needs to be figured out

Reply:
What do you guys think of fully filling the message that will be output as a single line at once?
Here is what I am suggesting at `SolverBase:950`:
```c++
      // get residual norms
      std::ostringstream convergenceState;
      residualNorm = calculateResidualNorm( time_n, stepDt, domain, m_dofManager, m_rhs.values(), convergenceState );
      convergenceState << GEOS_FMT( "        ( R ) = ( {:4.2e} )", residualNorm );
      GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::Convergence, convergenceState );
```
(`convergenceState` would be passed as reference in the `calculateResidualNorm()` calls, and filled just like the `std::cout` was filled before)

This way we definitively prevent this line to be "cut" if a warning appear.

Reply:
I would just create a macro without line break maybe